builder/metadata.go

```diff
@@ -151,16 +151,64 @@ func (b *Builder) getUpstreamFormatRange(version string) (format string, first,
 	return format, first, latest, err
 }
 
+// replaceInfoBundles wipes the <bundle>-info files and replaces them with
+// <bundle>/ directories that are empty. When the manifest creation happens it will
+// mark all files in that bundles as deleted.
+func (b *Builder) ReplaceInfoBundles(filenames []string) error {
+	for _, f := range filenames {
+		bdir := filepath.Join(b.Config.Builder.ServerStateDir, "image", b.MixVer, f)
+		// This bundle is deprecated, remove the info file
+		if err := os.RemoveAll(bdir + "-info"); err != nil {
```

Comment:
Can you just do an `os.Remove` related to the bug I pointed out in an earlier review?